src/components/Cards/Cards.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import classes from './Cards.module.scss'
+
+
+interface CardDataType {
+    startTask: {
+        hour: number,
+        minute: number
+    }
+    finishTask: {
+        hour: number,
+        minute: number
+    }
+    title: string
+    color: string 
+}
+
+export const Cards: React.FC<CardDataType> = ({ startTask, finishTask, title, color }) => {
```

Comment:
Setup `Cards` component interface. And I'd suggest renaming it to `Card` , because that's a single card.